pipelines/nextflow/modules/database/db_factory.nf

```diff
@@ -15,7 +15,7 @@
 
 process DB_FACTORY {
     tag "DB_factory"
-    label 'default'
+    label 'local'
```

Comment:
Out of curiosity, why do you want to run this module locally?

Reply:
Because it's lightweight and way too short to need a full job to be run